src/lib/containers/download_list_v2/DownloadListTable.tsx

```diff
@@ -106,102 +120,142 @@ export default function DownloadListTable() {
       )
     )
   }
+  const availableFiltersArray: AvailableFilter[] = [undefined, 'eligibleForPackaging', 'ineligibleForPackaging']
   return (
     <>
+      <div className="filterFilesContainer">
+          <span className="filterFilesByText">Filter Files By</span>
+          <Dropdown>
+            <Dropdown.Toggle variant="gray-primary-500" id="dropdown-basic">
+              {getFilterDisplayText(filter)}
+            </Dropdown.Toggle>
+            <Dropdown.Menu role="menu">
+            {availableFiltersArray.map(
+              (availableFilter) => (
+              <Dropdown.Item
+                  role="menuitem"
+                  key={`${getFilterDisplayText(availableFilter)}-filter-option`}
+                  onClick={() => {
+                    setFilter(availableFilter)
+                  }}
+                >
+                  {getFilterDisplayText(availableFilter)}
+                </Dropdown.Item>),)}
+            </Dropdown.Menu>
+          </Dropdown>
+      </div>
       {allRows.length > 0 && (
-        <ReactBootstrap.Table
```

Comment:
Rearranged because the filter UI should always visible (whether rows are returned or not, since we may have filtered them out)